gen-bundle-image.sh

```diff
@@ -57,7 +53,7 @@ function generate_manifest {
    local preset=$2
    podman manifest rm ${preset}-bundle:${version} || true
    podman manifest create ${preset}-bundle:${version}
-   if [[ ${preset} = "podman" ]]; then
+   if [[ ${preset} != "okd" ]]; then
```

Comment:
... but this part seems to be fixing a slightly different issue, the arm64 bundle was only added to a manifest/signed for the podman bundle, even if arm64 bundles are also generated for openshift since there is this code before:
```
   if [[ ${preset} = "openshift" ]]; then
       vfkit_bundle_arm64=crc_vfkit_${version}_arm64.crcbundle
   fi
```

Can this be moved to a different commit, or at least described in the commit log?

Reply:
Separate commit added for it.